<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushB">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>garland</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>95</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="openB">
       <property name="text">
        <string>open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeB">
       <property name="text">
        <string>close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>194</width>
      <height>182</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="redB">
       <property name="text">
        <string>red led</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="briBoxR">
       <item>
        <property name="text">
         <string>0%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>70%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>80%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>90%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100%</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modBoxR">
       <item>
        <property name="text">
         <string>rainbow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rainbowG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bpm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>juggle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sinelon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>confetti</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rasta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>white</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>швидкість</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="speedBU">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="speedBD">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="upB">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>update</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
